packages/super-components/components/msc-billing-tile/src/msc-billing-tile/Contact.tsx

```diff
@@ -0,0 +1,104 @@
+import { h, Fragment } from '@stencil/core';
+import {
+  OdsThemeColorIntent,
+  OdsThemeTypographyLevel,
+  OdsThemeTypographySize,
+} from '@ovhcloud/ods-theming';
+import { OdsIconName, OdsIconSize } from '@ovhcloud/ods-core';
+
+export const ContactContent = (
```

Comment:
why not creating with a class ? (export class) like the other ?

and the name of the file msc-billing-contact ?